mock/api.js

```diff
@@ -297,7 +282,11 @@ export const mockSession = {
 export default {
   'GET /controllers/months': mockIndices,
   'POST /controllers/list': mockControllers,
-  'POST /datasets/list': mockResults,
+  'POST /datasets/list': (req, res) => {
```

Comment:
What if we moved this to a helper function to wrap the mocked results in an object? There may be cases in the future where this is required for other mocked responses.

Reply:
fixed!